science/gildas/Portfile

```diff
@@ -30,8 +30,8 @@ master_sites        http://www.iram.fr/~gildas/dist/ \
                     http://www.iram.fr/~gildas/dist/archive/gildas/
 distname            ${name}-src-${my_version}
 
-checksums           rmd160  ca1ec4aef259cda4e896110ece75d7ddb14d7db6 \
-                    sha256  e95edfcda46d8976d6dd2839578b95d0ec335fc69cf5e6b250ebb533b606f72c
+checksums           rmd160  fd20e215480cc4339f4200085dfdfbdbf85ca1bf \
+                    sha256  3c747d90fbd30e2e1b9d94babfd3da3b896025d814967390aee62b5bac8120f6
 
```

Comment:
Please add the `size` tag to checksums.

Reply:
Can you please give some reference? I do not see such tag on the official documentation for Portfiles: https://guide.macports.org/chunked/development.creating-portfile.html

Reply:
It is the size of the download file in bytes. You can look at other Portfiles that use it like `py-numpy`. To calculate the value, just set it to 1 initially and use `port -v checksum` to get the correct value.